vinaigrette.py

```diff
@@ -0,0 +1,29 @@
+import datetime
+import random
+
+
+def random_date_in_range():
```

Comment:
whoa, how come all this logic is in a one function? each function should do only one thing

Reply:
split this code into multiple functions